test/unit/modules/keyboard.coffee

```diff
@@ -80,5 +80,20 @@ describe('Keyboard', ->
       expect(dom($('.ql-bold').get(0)).hasClass('ql-active')).toBe(true)
       expect(dom($('.ql-size').get(0)).value()).toBe(size)
     )
+
+    it('removeHotkey by name', ->
+      fn = @quill.getModule('keyboard').removeHotkey('BOLD')
+      expect(typeof fn).toBe('function');
+    )
+
+    it('removeHotkey by number', ->
```

Comment:
In my mind removeHotkey('enter') would almost always be preferable to removeHotkey(13). However you are right this is not possible right now since Keyboard.hotkeys does not define enter. Maybe it should just be added? `_initEnter` should be refactored at some point to use this but since it also overwrites shift enter I would just not worry about it right now.